.github/deployment/self-host/compose.yaml

```diff
@@ -0,0 +1,49 @@
+services:
+  affine:
+    image: ghcr.io/toeverything/affine-graphql:beta
+    container_name: affine
+    command:
+      [
+        'sh',
+        '-c',
+        './node_modules/.bin/dotenv -e /root/.affine/.env -- npm run predeploy && node --es-module-specifier-resolution=node ./dist/index.js',
+      ]
+    ports:
+      - '3010:3010'
+    depends_on:
+      redis:
+        condition: service_healthy
+      postgres:
+        condition: service_healthy
+    volumes:
+      - ~/.affine/storage:/root/.affine/storage
+      - ~/.affine/.env:/root/.affine/.env
+    environment:
+      - DISABLE_TELEMETRY=true
+      - NODE_ENV=production
+      - SERVER_FLAVOR=selfhosted
```

Comment:
consider add:
``` yaml
        logging:
            driver: "json-file"
            options:
                max-size: 10240k
```
to prevent large logs after long runing
and add
``` yaml
        restart: unless-stopped
```
to allow docker restart the container when restarting the server